plugins/events-backend-module-aws-sqs/README.md

```diff
@@ -32,7 +32,7 @@ events:
 
 ## Installation
 
-1. Install the [`events-backend` plugin](../events-backend/README.md).
+1. Install the [`events-node` package](../events-node/README.md).
```

Comment:
Still the `events-backend` right?

Reply:
No, `events-backend` is not needed anymore for the new setup. It only provides the `eventsPlugin` that adds the `/api/events/` routes incl. extension point.

This module only requires the service. That applies to many modules now.